File: app/src/main/java/com/project/scratchstudio/kith_andoid/network/model/comment/Comment.java
package com.project.scratchstudio.kith_andoid.network.model.comment;

import com.google.gson.annotations.SerializedName;
import com.project.scratchstudio.kith_andoid.network.model.user.User;

public class Comment {

    @SerializedName("message")
    private String message;

    @SerializedName("timestamp")
    private String timestamp;

    @SerializedName("user")
    private User user;

    public String getMessage() {
        return message;
    }

    public String getTimestamp() {
        return timestamp;
    }

    public User getUser() {
        return user;
    }
}

File: app/src/main/java/com/project/scratchstudio/kith_andoid/network/model/comment/Comment.kt
package com.project.scratchstudio.kith_andoid.network.model.comment

import com.google.gson.annotations.SerializedName
import com.project.scratchstudio.kith_andoid.network.model.user.User

class Comment {

    @SerializedName("id")
    var id = 0

    @SerializedName("message")
    var message = ""

    @SerializedName("timestamp")
    var timestamp = ""

    @SerializedName("user")
    lateinit var user: User

    @SerializedName("board")
    var boardId = 0
}
